From 34c244c910c7fb17cf600776b5a8b782b89d9f0e Mon Sep 17 00:00:00 2001
From: Sanyog Kale <skale@habana.ai>
Date: Tue, 28 Dec 2021 16:44:05 +0200
Subject: [PATCH] [SW-67662] gaudi2: add local otp fit as part of security package

This patch adds local otp fit as part of security package. This
otp fit can be used to perform signature verification for
fits signed with local keys when fake security is enabled.

Change-Id: Ia431a5b5941ae5f6319b0637858177a00db50930
---

diff --git a/scripts/make/gaudi2/Makefile b/scripts/make/gaudi2/Makefile
index ec33d6d..dd8bfdd 100644
--- a/scripts/make/gaudi2/Makefile
+++ b/scripts/make/gaudi2/Makefile
@@ -70,7 +70,8 @@
 	$(IMAGES_DIR)/${HL_DEV}-security_enable.itb		\
 	$(IMAGES_DIR)/${HL_DEV}-security_status.itb		\
 	$(IMAGES_DIR)/${HL_DEV}-update_svn.itb			\
-	$(IMAGES_DIR)/${HL_DEV}-otp.itb
+	$(IMAGES_DIR)/${HL_DEV}-otp.itb				\
+	$(IMAGES_DIR)/${HL_DEV}-otp-local.itb
 
 
 .PHONY: gaudi2-security-fit-create
